internal/aws/proxy/server.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws/endpoints"
 	v4 "github.com/aws/aws-sdk-go/aws/signer/v4"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/sanitize"
```

Comment:
I think you need a replace in go.mod?

Reply:
Already there https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/ab28decc091a8b49edae10121428d5be46eeac06/internal/aws/proxy/go.mod#L23